### DISCUSSION TITLE
ExampleNode not available for create?

### DISCUSSION THREAD

**Discussion by ndbn** [godot-zig/godot-zig#2] [General]
Hello, project work with Godot 4.3dev3, bindings generated and if open godot project ExampleNode will be under root Demo node at scene, but, not sure, is it normal that there are no ExampleNode inside Create New Node dialog?

Also can suggest to use above construction instead just return, created elements shown in editor
```zig
if (Godot.Engine.getSingleton().is_editor_hint()) {
        // return;
        self.set_process_mode(Godot.Node.PROCESS_MODE_DISABLED);
    }
```

![image](https://github.com/godot-zig/godot-zig/assets/2904562/6fe40427-1fcf-4e5d-b611-ae40e0dfd24f)

**Comment by elvisxzhou** [comment]
Thx for trying it out, I've fixed the issue by just setting GDExtensionClassCreationInfo2.is_exposed = 1, which is supposed to be that by default
